lib/dry/struct/setters.rb

```diff
@@ -27,6 +27,7 @@ def attributes(new_schema)
       end
 
       def self.define_setter_for(struct:, attribute:, type:)
+        attribute = Dry::Struct::Setters.remove_trailling_question_mark(attribute)
```

Comment:
No need for `Dry::Struct::Setters` here, `remove_trailling_question_mark(attribute)` will just work.

Reply:
And it should be `trailing` with a single `l`, right?